software/apps/audio_module/event_detector/Makefile

```diff
@@ -0,0 +1,17 @@
+# makefile for user application
+
+# the current directory
+APP_DIR := $(dir $(abspath $(lastword $(MAKEFILE_LIST))))
+
+# files needed for this code
+C_SRCS   := $(wildcard *.c) 
```

Comment:
Hi Amit,
Thank you so much for your help! I tried 
```
C_SRCS   := $(wildcard *.c)
C_SRCS   := $(wildcard **/log2fix.c)
```
but still get
```
DEP        log2fix/log2fix.c
<built-in>: fatal error: opening dependency file /mnt/c/Users/Long/Projects/signpost/software/apps/audio_module/event_detector/build/cortex-m4/log2fix/log2fix.d: No such file or directory
compilation terminated.
```
Namely, the Makefile still looks under the `build/` but didn't copy the source files into it?